packages/cashier/src/Components/Error/__tests__/funds-protection.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import FundsProtection from '../funds-protection.jsx';
```

Comment:
```suggestion
import FundsProtection from '../funds-protection';
```

Reply:
removed extra extension, thx for notice!